environment.yml

```
@@ -3,7 +3,9 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
-  - python>=3.10
+  # TODO: hyp3lib fails to import `cgi` which has been removed in Python 3.13: https://docs.python.org/3/library/cgi.html
+  #   we should decide how to resolve this
+  - python>=3.10,<3.13
```

We can revert this pin after https://github.com/ASFHyP3/hyp3-lib/pull/311 is released
```suggestion
  #   we should decide how to resolve this
  - python>=3.10
```